ConsistencyManager/Protocols/ConsistencyManagerModel.swift

```diff
@@ -188,3 +205,15 @@ public extension ConsistencyManagerModel where Self: Equatable {
         }
     }
 }
+
+public extension ConsistencyManagerModel {
+    func mergeModel(model: ConsistencyManagerModel) -> ConsistencyManagerModel? {
+        // Usually, we don't need to merge and instead just return the other model
```

Comment:
Will there be more code/implementation in this method body for the "unusual" case? Or for this file is it always just the model that we return?